otx/cli/utils/tests.py

```diff
@@ -15,12 +15,14 @@
 import json
 import os
 import shutil
+import yaml
 from subprocess import run  # nosec
 
 import cv2
 import numpy as np
 import pytest
 
+from openvino.runtime import Core
```

Comment:
Is that new dependency specified somewhere in requirement ?

Reply:
openvino is not in requirement? 
Which requirement i should change?

Reply:
In otx openvino should be installed together with cli, but in the legacy cli has it's own requirements file:
https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/ote_cli/requirements.txt